WcaOnRails/app/views/competitions/_competition_info.html.erb

```diff
@@ -39,7 +39,7 @@
         <dd><%=md competition.contact %></dd>
       <% end %>
 
-      <% if competition.base_entry_fee %>
+      <% if competition.base_entry_fee > 0 %>
```

Comment:
I must point out that this could be `if competition.base_entry_fee.nonzero?`.
That said, whatever you prefer ^^